data/services.json

```diff
@@ -165,16 +165,16 @@
+        "icon": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48c3ZnIGlkPSJf5ZyW5bGkXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHZpZXdCb3g9IjAgMCA0NzYuNDUgNTM2LjA5Ij48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6bm9uZTt9LmNscy0ye2NsaXAtcGF0aDp1cmwoI2NsaXBwYXRoKTt9LmNscy0ze2ZpbGw6IzA3Zjt9LmNscy00e2ZpbGw6IzE0YWFmZjt9PC9zdHlsZT48Y2xpcFBhdGggaWQ9ImNsaXBwYXRoIj48cmVjdCBjbGFzcz0iY2xzLTEiIHdpZHRoPSI0NzYuNDUiIGhlaWdodD0iNTM2LjA5Ii8+PC9jbGlwUGF0aD48L2RlZnM+PGcgY2xhc3M9ImNscy0yIj48Zz48cGF0aCBjbGFzcz0iY2xzLTQiIGQ9Ik0yNDcuMjYsMTk0LjI4Yy0yOC40NSwwLTU2LjIyLDEwLjk3LTc2Ljk4LDMwLjQzLTIyLjc3LDIxLjM1LTM3LjE4LDUyLjg4LTQ1LjQ0LDgyLjU0LTUuNDEsMTkuNDEtOC4xMiwzOS41NC04LjEyLDU5LjY4LDAsNi4wOSwuMjUsMTIuMTMsLjczLDE4LjEsLjU5LDcuMjYsNy42NiwxMi4yOSwxNC43LDEwLjM5LDYuMTQtMS42NSwxMi42LTIuNTQsMTkuMjYtMi41NCwxMy42OCwwLDI2LjUsMy43MiwzNy40OSwxMC4yMiwuMjcsLjE2LC41MywuMzIsLjgsLjQ3LDE3Ljg2LDEwLjY1LDM4LjkzLDE2LjUsNjEuNCwxNS43NSw1OC42MS0xLjk0LDEwNi40OS00OS42MywxMDguNjUtMTA4LjIzLDIuMzYtNjQuMTEtNDguOS0xMTYuODItMTEyLjQ4LTExNi44MlptMCwxNjMuOTljLTI4LjQxLDAtNTEuNDQtMjMuMDMtNTEuNDQtNTEuNDNzMjMuMDMtNTEuNDQsNTEuNDQtNTEuNDQsNTEuNDQsMjMuMDMsNTEuNDQsNTEuNDQtMjMuMDMsNTEuNDMtNTEuNDQsNTEuNDNaIi8+PHBhdGggY2xhc3M9ImNscy0zIiBkPSJNMjAwLjk4LDE1OC44NGgwYzAsMTQuODMtNy43NywyOC41OS0yMC41MywzNi4xNS04LjAxLDQuNzUtMTUuNTIsMTAuMzYtMjIuMzIsMTYuNzUtMTUuMDksMTQuMTQtMjYuMzUsMzEuNDktMzQuNjksNDguNjktMS42NCwzLjM5LTYuNzMsMi4xOS02LjczLTEuNTh2LTEwMC4wMWMwLTIzLjI3LDE4Ljg3LTQyLjE0LDQyLjE0LTQyLjE0aDBjMjMuMjcsMCw0Mi4xNCwxOC44Nyw0Mi4xNCw0Mi4xNFoiLz48L2c+PC9nPjwvc3ZnPg==",
         "description": "Your entrance to the blockchain world.",
-        "color": "#afd8f7",
+        "color": "#0075FF",
         "supportEmail": "support@blocto.app",
-        "authn_endpoint": "https://flow-wallet-testnet.blocto.app/authn",
-        "website": "https://blocto.portto.io"
+        "authn_endpoint": "https://wallet-v2-dev.blocto.app/-/flow/authn",
```

Comment:
Only flag is that this endpoint is technically incorrect (when I used it, it was along the lines of `https://wallet-v2-dev.blocto.app/00000000-0000-0000-0000-000000000000/flow/authn`).  However, I don't think this config key is used for anything, so I'm wondering if we should just scrap it.